visualization_msgs/Marker box_scale
geometry_msgs/Pose box_pose